doc/langref.html.in

```diff
@@ -2861,6 +2861,37 @@ fn dump(args: anytype) void {
     expect(args.b);
     expect(args.s[0] == 'h');
     expect(args.s[1] == 'i');
+}
+      {#code_end#}
+      <p>
+      Anonymous structs can be created without specifying field names, and are referred to as "tuples".
+      </p>
+      <p>
+      The fields are implicitly named using numbers starting from 0. Because their names are integers,
+      the {#syntax#}@"0"{#endsyntax#} syntax must be used to access them. Names inside {#syntax#}@""{#endsyntax#} are always recognised as identifiers.
```

Comment:
We will have to add a note here for #8262

Reply:
You are confusing ascii 0 with the null byte. And also struct accessors don't show up in the binary (they are just offsets), so the restriction does not show up here.

Reply:
> You are confusing ascii 0 with the null byte.

I am not. This is not about `@"0"`.
I am just saying that we should add a note in this section about that peculiar issue.